lambda_src/lambda_function.py

```diff
@@ -97,9 +126,9 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
     LOG.debug('Destination header not found in a POST and hence using sync_flow().')
     headers = event['headers']
     req_body = loads(event['body'])
-
+    write_uri = headers.get('sf-custom-write-uri')
```

Comment:
I think this might have intentionally been set to something the user could not pass, but I might be mis-remembering. why the change?

Reply:
This was so that we could format the `write-uri`, as the destionation header is popped earlier and `process_row_params_format` filters on `sf-custom`

Reply:
Removed the `sf-custom`  from `write-uri` and used a `format` like [here](https://github.com/Snowflake-Labs/geff/pull/64#discussion_r1116158870). 

https://github.com/Snowflake-Labs/geff/blob/7da2fef21098b2ed3a522aeea92ac8fb20d0c6f9/lambda_src/lambda_function.py#L108-L126